compute_endpoint/globus_compute_endpoint/MULTIUSER_SETUP.md

```diff
@@ -0,0 +1,334 @@
+   Slack](https://funcx.slack.com/archives/C017637NZFA).
+
+1. Modify `user_config_template.yaml` as appropriate for the resources you want
+   to make available.  This file will be interpreted as a Jinja template, and
+   will become, after template processing with the user-provided variables, the
+   configuration that user endpoints will utilize.  The default configuration
+   (as created in step 4) has a basic working configuration, but uses the
+   `LocalProvider`.
+
+   Please look to [the documentation for a number of known working
+   examples](https://globus-compute.readthedocs.io/en/latest/endpoints.html#example-configurations)
+   (all written for single-user use) as a starting point.
+
+1. Modify `user_environment.yaml` for any environment variables you would like
+   injected into user endpoints process space:
+
+    ```yaml
+    SOME_SITE_SPECIFIC_ENV_VAR: a site specific value
+    ```
+
+1. Run MEP manually for testing and easier debugging, as well as to collect the
+   (Multi-User) endpoint ID for sharing with users.  The first time through,
+   the Globus Compute Endpoint will initiate a Globus Auth login flow, and
+   present a long URL:
+
+    ```console
+    # globus-compute-endpoint start prod_gpu_large
+    > Endpoint Manager initialization
+    Please authenticate with Globus here:
+    ------------------------------------
+    https://auth.globus.org/v2/oauth2/authorize?clie...&prompt=login
+    ------------------------------------
+
+    Enter the resulting Authorization Code here: <PASTE CODE HERE AND PRESS ENTER>
+    ```
+
+1. While iterating, the `--log-to-console` flag may be useful to emit the log
+   lines to the console (also available at
+   `.globus_compute/prod_gpu_large/endpoint.log`).
+
+    ```console
+    # globus-compute-endpoint start prod_gpu_large --log-to-console
+    > Endpoint Manager initialization
+    >
+
+    ========== Endpoint Manager begins: 1ed568ab-79ec-4f7c-be78-a704439b2266
+            >>> Multi-User Endpoint ID: 1ed568ab-79ec-4f7c-be78-a704439b2266 <<<
+    ```
+
+    Additionally, for even noiser output, there is `--debug`.
+
+1. When satisfied, use a `systemd` unit file to automatically start the
+   multi-user endpoint when the host boots:
+
+    ```console
+    # cat /etc/systemd/system/globus-compute-endpoint-prod_gpu_large.service
+    [Unit]
+    Description=Globus Compute Endpoint systemd service (prod_gpu_large)
+    After=network.target
+    StartLimitIntervalSec=0
+
+    [Service]
+    ExecStart=/opt/globus-compute-endpoint/bin/globus-compute-endpoint start prod_gpu_large
+    User=root
+    Type=simple
+    Restart=always
+    RestartSec=30
+
+    [Install]
+    WantedBy=multi-user.target
+    ```
+
+    And enable via the usual interaction:
+
+    ```console
+    # systemctl enable globus-compute-endpoint-prod_gpu_large --now
+    ```
+
+## Background
+
+A multi-user endpoint (MEP) might be thought of as a user-endpoint (UEP)
```

Comment:
Consider MUEP and SUEP (single-user endpoint) instead of MEP and UEP.

Reply:
Without a larger reasoning behind this suggestion, I'm inclined to stick with the precedent that was set in the ParslFest 2023 presentation (MEP and UEP).

Reply:
Precedent is a good reason to keep it as-is. My suggestion was only a preference.